### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MarkMatiushechko**
- [DEMO LINK](https://MarkMatiushechko.github.io/layout_landing-page/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,170 @@
...
+  left: 0;
+
+  opacity: 0;
+  pointer-events: none;
+}
+
+.page__menu:target {
+  opacity: 1;
+  pointer-events: all;
+}
```
Comment (l4st1m0za) [inline src/styles/main.scss:28]: It's better to use nesting as you have SCSS file which alows it. Try to implement it in all the possible places.

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,170 @@
-body {
+// #region page
+.page {
+  font-family: Manrope, sans-serif;
+  color: #1B2129;
```
Comment (l4st1m0za) [inline src/styles/main.scss:4]: It's better not to use single colour name, but to use constant which represents the colour.

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Great job! See the required changes. Also, it's better to use **.svg** instead of **.png** images. Continue working!

**Comment by MarkMatiushechko** [comment]
[DEMO LINK](https://MarkMatiushechko.github.io/layout_landing-page/)

**Pull request closed without merging**